### ISSUE TITLE
bug: `config.devtools` doesn't provide a remote tunnel URL

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1585]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.10.15](https://github.com/open-wa/wa-automate-nodejs/compare/3.10.14...3.10.15) (2021-04-14)

- build(deps-dev): bump @typescript-eslint/parser from 4.21.0 to 4.22.0 [`#1582`](https://github.com/open-wa/wa-automate-nodejs/pull/1582)
- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.21.0 to 4.22.0 [`#1583`](https://github.com/open-wa/wa-automate-nodejs/pull/1583)
- build(deps-dev): bump @types/marked from 2.0.1 to 2.0.2 [`#1584`](https://github.com/open-wa/wa-automate-nodejs/pull/1584)
- 🐛  fix #1585 [`#1585`](https://github.com/open-wa/wa-automate-nodejs/issues/1585)
- Release 3.10.15 [`269c850`](https://github.com/open-wa/wa-automate-nodejs/commit/269c8504afb1f81ea629ee3ce2a9a03a8182fbae)
